regions_www/m/server.R

```diff
@@ -417,8 +417,13 @@ shinyServer(function(input, output, session) {
       if (all(!keep))
         return(NULL)
       lines_in_bb <- lines[drop(keep),]
-      # Sort by the absolute values
-      lines_in_bb[tail(order(abs(lines_in_bb[[line_data()]])), nos),]
+      # Sort low-to-high for reduction in deaths (can't use absolute values as no. deaths can be a positive number, i.e. health disbenefit)
+      if (grepl(c("sideath_heat"), line_data())) {
```

Comment:
We don't let people search for `sivalue_heat` but that can also be -ve or +ve, just checking we've no intention of adding that.  If not than :+1: from me, really nice spot.